x-pack/legacy/plugins/ml/public/data_frame/pages/transform_management/components/transform_list/action_delete.tsx

```diff
@@ -24,11 +24,21 @@ import {
 import { DataFrameTransformListRow, DATA_FRAME_TRANSFORM_STATE } from './common';
 
 interface DeleteActionProps {
-  item: DataFrameTransformListRow;
+  item?: DataFrameTransformListRow;
```

Comment:
Having all props optional means this could be used with neither.
Rather than having two optional props, it could only take `items` and `isBulkAction` is then determined based on the length of the `items` list.

Reply:
Yeah this was a tricky one to decide how to Type. It needs to have either one or the other. I wasn't sure how to Type it so that it requires at least one of those. 🤔 
Unless you mean put the single item in an array and only just expect the one `items` argument.

Reply:
> Unless you mean put the single item in an array and only just expect the one items argument.

Yes, exactly that.

Reply:
Great call - always pass an array of items which allows us to correctly define param type and also simplifies the way we call the endpoint. Updated in 7988c5d5c1077aaeb9f527ff48928e3019bd692e